syft/serde/protobuf/serde.py

```diff
@@ -401,22 +401,27 @@ def bufferize_arg(worker: AbstractWorker, arg: object) -> ArgPB:
         setattr(protobuf_arg, attr_name, arg)
     except:
         getattr(protobuf_arg, attr_name).CopyFrom(_bufferize(worker, arg))
+
     return protobuf_arg
 
 
 def unbufferize_args(worker: AbstractWorker, protobuf_args: list) -> list:
     args_ = []
     for protobuf_arg in protobuf_args:
-        args_.append(unbufferize_arg(worker, protobuf_arg))
-    return args_
+        unbufferized = unbufferize_arg(worker, protobuf_arg)
```

Comment:
I would have done it with a list comprehension here, but it's a detail

Reply:
You're right, that's cleaner. Fixing.